modules/configuring-localnet-switched-topology.adoc

```diff
@@ -6,22 +6,103 @@
 [id="configuration-localnet-switched-topology_{context}"]
 = Configuration for a localnet switched topology
 
+// To accommodate a link to the NMstate Operator, the content in this module
+// is split with tags. The tag includes don't pull in the module header above.
+
+// tag::localnet-intro[]
 The switched (localnet) topology interconnects the workloads through a cluster-wide logical switch to a physical network.
+// end::localnet-intro[]
+
+// tag::localnet-content[]
+You must map the additional network to the OVN bridge to use it as an OVN-Kubernetes additional network. Bridge mappings allow network traffic to reach the physical network. A bridge mapping associates a physical network name, also known as an interface label, to a bridge created with OVS or OVN.
+
+You can create an `NodeNetworkConfigurationPolicy` object, part of the `nmstate.io/v1` API group, to declaratively create the mapping. This API is provided by the NMState Operator. By using this API, you can apply the bridge mapping to nodes that match the `nodeSelector` that you specify, such as `node-role.kubernetes.io/worker: ''` to apply to all worker nodes in your cluster.
+
+When attaching an additional network, you can either use the existing `br-ex` bridge or create a new bridge. Which approach to use depends on your specific network infrastructure.
+
+- If your nodes include only a single network interface, you must use the existing bridge. This network interface is owned and managed by OVN-Kubernetes and you must not remove it from the `br-ex` bridge or alter the interface configuration, or your cluster network will stop working correctly.
+- If your nodes include several network interfaces, you attach a different network interface to a new bridge, and use that for your additional network. This approach provides for traffic isolation from your primary cluster network.
+
+In the following example, the `localnet1` network is mapped to the `ovs-ex`:
```

Comment:
I think this is wrong; don't you mean to point this physical network to the `br-ex` bride ? 

```suggestion
In the following example, the `localnet1` network is mapped to the `br-ex` OVS bridge:
```